cli.js

```diff
@@ -142,17 +208,29 @@ async function measureUpload(bytes, iterations) {
   return measurements;
 }
 
+function logInfo(text, data) {
+  console.log(bold(' '.repeat(15 - text.length), `${text}:`, blue(data)));
+}
+
 function logSpeedTestResult(size, test) {
   const speed = stats.median(test).toFixed(2);
   console.log(
-    bold(' '.repeat(7 - size.length), size, 'speed:', yellow(`${speed} Mbps`))
+    bold(' '.repeat(9 - size.length), size, 'speed:', yellow(`${speed} Mbps`))
   );
 }
 
 async function speedTest() {
+  const serverLocationData = await fetchServerLocationData();
+
+  const { ip, loc, colo } = await fetchCfCdnCgiTrace();
+
+  logInfo('Your IP', ip);
```

Comment:
`Your IP` -> `Your network` to be consistent with CF, also it looks better IMO

Reply:
Cloudflare use "Your Network" because it provided ASN information (which we don't have).

Reply:
Yeah it may be confusing